src/Logins/PatientLogin.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useState, useEffect } from "react";
+import "./Patientlogin.css"
+
+function PatientLogin() {
+  const initialValues = { username: "", email: "", password: "" };
+  const [formValues, setFormValues] = useState(initialValues);
+  const [formErrors, setFormErrors] = useState({});
+  const [isSubmit, setIsSubmit] = useState(false);
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormValues({ ...formValues, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setFormErrors(validate(formValues));
+    setIsSubmit(true);
```

Comment:
If there are errors why are we letting them submit? This should only be set to true if there are no errors.